config/dagu/scripts/git-maintenance.py

```
@@ -0,0 +1,265 @@
+# /// script
```

Ruff (0.15.7)

[warning] The file is executable but no shebang is present

(EXE002)